### DISCUSSION TITLE
Setting shipping information carrier/tracking

### DISCUSSION THREAD

**Discussion by HDPAlex** [amzn/selling-partner-api-models#3249] [General]
Hello
How to upload shipping info for an order?
updateShipmentStatus operation changes only status

No way to upload carrier/tracking info
Currently use _POST_ORDER_FULFILLMENT_DATA_ on MWS
Can updateShipmentStatus  take optional values for carrier and tracking number?

**Comment by mulloa88** [comment]
I believe this is what the PUT Transportation Details API is for. Below is an example of non-partnered small parcel carrier:

https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v0-reference#nonpartneredsmallparcelpackageinputlist

**Comment by HDPAlex** [comment]
Transportation detail are for the fulfillment shipments
I am looking for an easy way for manufacturer filled orders
Hoped to avoid uploading feeds

**Comment by SmokinB** [comment]
That's also what I'm looking for.
Mark orders as shipped and provide tracking number.
Any news here?

**Reply by HDPAlex** [reply]
Hello

There is no other way to do it but to upload a POST_ORDER_FULFILLMENT_DATA feed
At least I could not find it

**Reply by SmokinB** [reply]
Thank you @HDPAlex
Yes, I ended up sending a "POST_FLAT_FILE_FULFILLMENT_DATA" feed.
Very frustrating API + Documentation...

**Comment by ivancortesromero** [comment]
@HDPAlex, I'm having the same issue today. How did you manage it? My current approach is using [confirmShipment](https://developer-docs.amazon.com/sp-api/docs/orders-api-v0-reference#post-ordersv0ordersorderidshipmentconfirmation), did you use it?

**Comment by HDPAlex** [comment]
We finally decided to use order feed POST_ORDER_FULFILLMENT_DATA so we can confirm multiple shipments at at one time
